### PR TITLE
Add lightweight evasive transforms to evade rejection false positives

### LINKED ISSUE #498: import expression false positives
we're encountering a lot of issues with false positives in the detection of `import()` statements when evaluating code from npm

> possible import expression rejected around line {number}

I'll use my lavamoat-survey tool to get a list of offending code samples in popular packages on npm

### PR DESCRIPTION
**Pull request by erights**
Replaces #499 

Progress on #498.

Note that these are lightweight because they avoid parsing. As a result, they can change the meaning of some programs. Any system that parses and transforms should instead do its own accurate evasion by generating code that avoids the rejections without changing the meaning of programs. This PR has no examples or support for such accurate evasions.

### CODE REVIEW

File: packages/ses/ERRORS.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ also use a comma expression.
 (0, compartment.import)(specifier);
 ```
 
-## SES2: Possible HTML comment rejected
+## Possible HTML comment rejected
```
Comment (erights) [inline packages/ses/ERRORS.md:104]: With the links being to anchors in a page, when you follow a link it is not at all clear what section you're supposed to look at. Perhaps each of these targets should be its own standalone explanation within a subdirectory of such explanations.

File: packages/ses/src/transforms.js [hunk 2]
```diff
@@ -12,111 +19,246 @@ function getLineNumber(src, pattern) {
...
 export function rejectHtmlComments(src) {
   const lineNumber = getLineNumber(src, htmlCommentPattern);
   if (lineNumber < 0) {
     return src;
   }
   const name = getSourceURL(src);
   throw new SyntaxError(
-    `SES3: Possible HTML comment rejected at ${name}:${lineNumber}`,
+    `Possible HTML comment rejected at ${name}:${lineNumber}
+    See https://github.com/Agoric/SES-shim/blob/evade-rejections/packages/ses/ERRORS.md#possible-html-comment-rejected`,
```
Comment (erights) [inline packages/ses/src/transforms.js:61]: Needs to be restored to `master` branch before merging. Which raises an interesting issue. I don't see any simple way to keep the links relative to the branch during development. This argues that the prefix should come from somewhere else, which is more consistent with the #545 keyword experiment.

File: packages/ses/src/transforms.js [hunk 3]
```diff
@@ -12,111 +19,246 @@ function getLineNumber(src, pattern) {
...
 export function rejectImportExpressions(src) {
   const lineNumber = getLineNumber(src, importPattern);
   if (lineNumber < 0) {
     return src;
   }
   const name = getSourceURL(src);
   throw new SyntaxError(
-    `SES2: Possible import expression rejected at ${name}:${lineNumber}`,
+    `Possible import expression rejected at ${name}:${lineNumber}
+    See https://github.com/Agoric/SES-shim/blob/evade-rejections/packages/ses/ERRORS.md#possible-import-expression-rejected`,
```
Comment (erights) [inline packages/ses/src/transforms.js:134]: Restore to `master` branch.

File: packages/ses/src/transforms.js [hunk 4]
```diff
@@ -12,111 +19,246 @@ function getLineNumber(src, pattern) {
...
 export function rejectSomeDirectEvalExpressions(src) {
   const lineNumber = getLineNumber(src, someDirectEvalPattern);
   if (lineNumber < 0) {
     return src;
   }
   const name = getSourceURL(src);
   throw new SyntaxError(
-    `SES1: Possible direct eval expression rejected at ${name}:${lineNumber}`,
+    `Possible direct eval expression rejected at ${name}:${lineNumber}
+    See https://github.com/Agoric/SES-shim/blob/evade-rejections/packages/ses/ERRORS.md#possible-eval-expression-reject`,
```
Comment (erights) [inline packages/ses/src/transforms.js:207]: Restore to `master` branch.

**Comment by erights** [comment]
Ready for review.

Notice on own comments. This is an experiment that may well be a bad idea we should not merge. Looking for feedback.

File: packages/ses/ERRORS.md (hunk 1 above)
Comment (erights) [reply packages/ses/ERRORS.md]: Changed each to a separate standalone page with an `error-codes` subdirectory. The page filename is now a pleasingly symbolic identifier without putting these identifiers in the programmer's face. Resolving.

**Review by kriskowal — APPROVED** [review]

**Pull request merged**